src/index.html

```diff
@@ -11,7 +11,228 @@
+              >
+            </label>
+          </div>
+          <div class="input">
+            <label for="birth">
+              Full date of birth:
+              <input
+                type="date"
+                name="date"
+                id="birth"
+              >
+            </label>
+          </div>
+          <div class="last">
+            <label for="term">
+              I accept the term of the agreement
+              <input
+                type="checkbox"
+                name="agree"
+                id="term"
+                class="last"
+                required
+              >
+            </label>
+          </div>
+        </fieldset>
+      </div>
+      <div class="field">
+        <fieldset>
+          <legend>Registration</legend>
+          <div class="input">
+            <label for="mail">
+              E-mail:
+              <input
+                type="email"
+                name="email"
+                id="mail"
+                placeholder="email@example.com"
+                required
+              >
+            </label>
+          </div>
+          <div class="last">
+            <label for="passw">
+              Password:
+              <input
+                type="password"
+                name="password"
+                id="passw"
+                class="last"
+                minlength="5"
+                required
+              >
+            </label>
+          </div>
+        </fieldset>
+      </div>
+      <div class="field">
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <div class="input">
+            <label for="yescats">
+              Do you love cats?
+              <input
+                type="radio"
+                name="cats"
+                id="yescats"
+              > Yes
+            </label>
+            <label for="nocats">
+              <input
+                type="radio"
+                name="cats"
+                id="nocats"
+              > No
+            </label>
+          </div>
+          <div class="input">
+            <label for="favcolor">
+              What is your favorite color?
+              <input
+                type="color"
+                name="color"
+                id="favcolor"
+              >
+            </label>
+          </div>
+          <div class="input">
+            <label for="bed">
+              What time do you go to bed?
+              <input
+                type="time"
+                name="time"
+                id="bed"
+                step="1"
+              >
+            </label>
+          </div>
+          <div class="input">
+            <label for="cars"
+            class="input">
+              What are your favorite brand of cars?
+              <select
+                name="brand"
+                id="cars"
+                multiple
+              >
+                <option value="BMW"
+                  name="BMW">BMW
+                </option>
```

Comment:
`name` attribute for option is useless.
```suggestion
                <option value="BMW">BMW</option>
```